entgo-sveltekit/template/package.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
This file has to be added to .github/dependabot.yml

Reply:
I made the fix.